### DISCUSSION TITLE
[V5] - Correct way to use UTC with Lucid

### DISCUSSION THREAD

**Discussion by sn0rcha** [adonisjs/core#1961] [Help]
Thanks for this new version of Adonis, it's awesome!

Quick question, what is the correct way to store timestamps in UTC with Lucid?

I've noticed that

```
  @column.dateTime({ autoCreate: true })
  public createdAt: DateTime

  @column.dateTime({ autoCreate: true, autoUpdate: true })
  public updatedAt: DateTime
```

Stores the value in the local timezone.

What is the best way to ensure that dates and times are stored and retrieved in UTC?

Cheers,
Ben

**Comment by thetutlage** [comment]
Run your DB server in UTC timezone? 

Getting the timezone right with database is tricky and depends a lot on the database you are using. PostgreSQL accepts the timezone offset and manage the conversion itself. Whereas older versions of MySQL doesn't have great support for it.

I you use PostgreSQL, then read this article. http://blog.untrod.com/2016/08/actually-understanding-timezones-in-postgresql.html

**Reply by sn0rcha** [reply]
Thanks for the info and the link, I realized my timestamp fields were not ```timestamptz``` - doh!
